runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -503,6 +547,24 @@ private void killedOne()
 		infoTimer = Instant.now();
 	}
 
+	private boolean doubleTroubleExtraKill()
+	{
+		return config.doubleTroubleEnabled() &&
+				config.taskName().equalsIgnoreCase(Task.GROTESQUE_GUARDIANS.name()) || config.taskName().equalsIgnoreCase(Task.GARGOYLES.name()) &&
+				isInGrotesqueGuardiansScene();
+	}
+
+	private boolean isInGrotesqueGuardiansScene()
+	{
+		Point minSceneLocation = new Point(41, 55);
```

Comment:
The fact these effectively equate to magic numbers doesn't sit well with me.

Also couldn't this be better achieved with regionID's?

Reply:
In this case, magic numbers seem more readable since replacing them with constants that don't have a really meaningful name seems a little obnoxious.

I tried the region ID first, but it doesn't stay the same because it's an instance. 6727 is what displays on the screen but when you check it in code (e.g. `client.getLocalPlayer().getWorldLocation().getRegionID()`) it varies, same with the X,Y coordinates. Let me know if you know of a better way to check a consistent location, though.

Reply:
`WorldPoint.fromLocalInstance().getRegionId()`